cruise-control-metrics-reporter/src/test/java/com/linkedin/kafka/cruisecontrol/metricsreporter/utils/CCContainerizedKraftCluster.java

```diff
@@ -185,14 +187,19 @@ private void copyCertToContainer(KafkaContainer container, Map<Object, Object> c
   }
 
   private void setupContainerResources(KafkaContainer kafkaContainer, Map<Object, Object> brokerConfig) {
-    Path libsDir = Paths.get("build", "libs").toAbsolutePath().normalize();
+    Path projectRoot = Paths.get("").toAbsolutePath().normalize();
+
+    Path metricsReporterJarPath = projectRoot
+      .resolveSibling("cruise-control-metrics-reporter")
+      .resolve("build")
+      .resolve("libs");
 
     try {
-      Path jarPath = Files.list(libsDir)
+      Path jarPath = Files.list(metricsReporterJarPath)
         .filter(path -> path.getFileName().toString().startsWith("cruise-control-metrics-reporter"))
         .filter(path -> path.getFileName().toString().endsWith(".jar"))
         .findFirst()
-        .orElseThrow(() -> new IllegalStateException("Cruise Control Metrics Reporter jar not found in: " + libsDir));
+        .orElseThrow(() -> new IllegalStateException("Cruise Control Metrics Reporter jar not found in: " + metricsReporterJarPath));
```

Comment:
Oh, it's just now I realize that the tests would rely on the (metrics reporter) project to be built beforehand. 

Does it make sense to mention this requirement in the class JavaDoc and/or in the exception's message?

Reply:
Yes definitely, just added.